Gifski/Tooltip.swift

```diff
@@ -0,0 +1,93 @@
+import AppKit
+
+final class Tooltip: NSPopover {
+	init(text: String, closeOnClick: Bool = true, contentInsets: NSEdgeInsets = .zero, maxWidth: CGFloat? = nil) {
```

Comment:
I think `contentInsets` should default to `10` and we should use the default in our tooltip. It's a good-looking default.

I believe in sensible defaults, so I think we should default `maxWidth` to `200`, but still accept `nil` in case the user doesn't want a default. The reason for 200 is that tooltips are meant to be small UI with not much text.